pkg/querier/chunk_store_queryable.go

```diff
@@ -103,6 +104,7 @@ func (s *chunkSeries) Iterator() storage.SeriesIterator {
 	return s.chunkIteratorFunc(s.chunks, model.Time(s.mint), model.Time(s.maxt))
 }
 
+// method from SeriesWithChunks interface
```

Comment:
Comments for exported methods usually start with the method names :) (just a Go convention)
```suggestion
// Chunks implements SeriesWithChunks interface.
```

Reply:
:-) Done!